misc/python/materialize/cli/gen-lints.py

```diff
@@ -46,6 +46,8 @@
     # and the suggested solution—boxing the large variant—might actually result
     # in slower code due to the allocation.
     ("large_enum_variant", 0),
+    # A specialization of `large_enum_variant`; similar arguments apply.
+    ("result_large_err", 0),
```

Comment:
I actually think enabling this lint might be useful. But fixing all the instances it finds is a bunch of effort and I didn't want to block the fix for the `std::sync::mpsc` safety bug on it.